astropy/io/fits/util.py

```diff
@@ -763,7 +763,7 @@ def _words_group(input, strlen):
     words = []
     nblanks = input.count(' ')
     nmax = max(nblanks, len(input) // strlen + 1)
-    arr = np.fromstring((input + ' '), dtype=(bytes, 1))
+    arr = np.frombuffer((input + ' ').encode('utf8'), dtype=(bytes, 1))
```

Comment:
This is quite ugly; really should rewrite the below without using an array. But that's beyond the scope of this PR...

Reply:
I was reading this while looking at other uses of `np.frombuffer`, and actually I wonder why this function exists instead of using `textwrap.wrap`... maybe some weird edge case that I don't see, but on a simple string (as tested with `test_long_string_value` it gives the same result).